daffodil-udf/src/main/java/org/apache/daffodil/udf/UserDefinedFunctionIdentification.java

```diff
@@ -28,11 +28,21 @@
  *
  * It must have the name and namespaceURI fields initialized with the namespace
  * and name callers would be expected to use in the schema.
- *
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
 public @interface UserDefinedFunctionIdentification {
+  /**
+   * Get the name of the user defined function
+   *
+   * @return the name of the user defined function
+   */
   String name();
+
+  /**
+   * Get the namespace URI of the user defined function
```

Comment:
Gaaak. My job is at the redundancy department is where I work.  :-)

I hate having to have 6 lines All of which basically say the same thing, and add no value over just the function signature itself.  To me this is just code pollution. Might as well say:
```
/**
 * Read the f***ing method name.  (RTFM)
 */
```

What happens if you have no javadoc for this at all? The javadoc generated will still show this as a public member right?

Reply:
I agree, this feels very unnecessary. Javadoc shows up just fine, the only downside is this will output a warning until you add some javadoc. I was trying to minimize the number of warnings so that it's easier to see when we are missing actually important javadoc that we're actually missing.

Reply:
Ok then. Will one-liners work. 
```
/** @returns the value */
```
Might work. 1 line vs. 5.

Reply:
I agree with removing redundancy, but I don't see much value in trying to save lines or even removing descriptions. It maybe feels verbose, but this is standard practice in Javadocs. And perhaps the real issue is this docs are just not good docs. For example, maybe this should be more along the lines of:

```scala
/**
 * Get the local name of the user defined function, excluding a namespace or prefix
 *
 * @returns the local name
 */
```

Which I think does provide some added value that just "name()" doesn't necessarily imply